app/views/admin/account_warnings/_account_warning.html.haml

```diff
@@ -5,7 +5,7 @@
         = fa_icon 'warning'
     .log-entry__content
       .log-entry__title
-        = t(account_warning.action, scope: 'admin.strikes.actions', name: content_tag(:span, account_warning.account.username, class: 'username'), target: content_tag(:span, account_warning.target_account.pretty_acct, class: 'target')).html_safe
+        = t(account_warning.action, scope: 'admin.strikes.actions', name: content_tag(:span, account_warning.account ? account_warning.account.username : 'Unknown Account (Deleted?)', class: 'username'), target: content_tag(:span, account_warning.target_account.pretty_acct, class: 'target')).html_safe
```

Comment:
Hi! Thank you for your contribution!

You should not use a harcoded string, but a translatable string instead (those are used with `t('translation_id')` and defined in `config/locales/en.yml`

Reply:
Hi! I ended up taking over and fixing it myself.